statsd/statsd.go

```diff
@@ -378,10 +380,10 @@ func createWriter(addr string, writeTimeout time.Duration) (io.WriteCloser, stri
 		return w, writerNameUDS, err
 	case strings.HasPrefix(addr, UnixAddressDatagramPrefix):
 		w, err := newUDSWriter(addr[len(UnixAddressDatagramPrefix):], writeTimeout, "unixgram")
-		return w, writerNameUDS, err
+		return w, writerNameUDSDatagram, err
```

Comment:
Why use a distinct name here, isn't the behavior the same as with unix socket with no scheme prefix?

Reply:
It is, but this allows the `client_transport` tag to be correctly set by in telemetry client